plugins/lookup/tss.py

```diff
@@ -57,6 +57,17 @@
               key: domain
         required: false
         version_added: 3.6.0
+    token:
+        description:
+          - Existing token for Thycotic authorizer.
```

Comment:
Here, something like:
```suggestion
          - Existing token for Thycotic authorizer.
          - If provided, I(username) and I(password) are not needed.
```